robotbase/src/main/java/com/team2357/frc2022/commands/human/ClimbProgressionCommand.java

```diff
@@ -1,15 +1,49 @@
 package com.team2357.frc2022.commands.human;
 
+import com.team2357.frc2022.commands.climb.ClimberClimbToReachableCommandGroup;
+import com.team2357.frc2022.commands.climb.ClimberClimbToRungCommandGroup;
 import com.team2357.lib.commands.CommandLoggerBase;
 
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
 public class ClimbProgressionCommand extends CommandLoggerBase {
+    private SequentialCommandGroup climbCommands[] = {
+            new ClimberClimbToReachableCommandGroup(),
+            new ClimberClimbToRungCommandGroup(),
+            new ClimberClimbToRungCommandGroup()
+    };
+
+    private int commandIndex;
+    private long holdTimeMs;
+
+    public ClimbProgressionCommand() {
+        commandIndex = -1;
+    }
+
     @Override
     public void initialize() {
-        System.out.println("ClimbProgressionCommand");
+        if (commandIndex > -1 && commandIndex < climbCommands.length) {
+            climbCommands[commandIndex++].schedule();
+        }
+
+        if (commandIndex == -1) {
```

Comment:
We could have this first in this function then return and not check for the -1 condition above.